src/SFA.DAS.QnA.Application/Commands/ResetPageAnswers/ResetPageAnswersBySectionNoHandler.cs

```diff
@@ -20,6 +18,7 @@ public ResetPageAnswersBySectionNoHandler(QnaDataContext dataContext, INotRequir
 
         public async Task<HandlerResponse<ResetPageAnswersResponse>> Handle(ResetPageAnswersBySectionNoRequest request, CancellationToken cancellationToken)
         {
+            var application = await _dataContext.Applications.SingleOrDefaultAsync(app => app.Id == request.ApplicationId, cancellationToken);
```

Comment:
I'd move this back to being below the validationErrorResponse  so you're not wasting a database call if validation fails

Reply:
Done